src/ui/compilationStatusView.ts

```diff
@@ -19,7 +19,20 @@ function toStatusMessage(status: CompilationStatus): string {
     return Messages.CompilationStatus.ResolutionFailed;
   case CompilationStatus.CompilationSucceeded:
     return Messages.CompilationStatus.CompilationSucceeded;
-  default: throw new Error(`Should not handle status message ${status}`);
+
+  // Not used from 3.8 on
```

Comment:
```suggestion
  // Backward compatibility for versions <= 3.7.X
```
Alternatively, we haven't decided the numbering of the new version of Dafny, whether it'll be 3.8 or 3.7.4. so for now I'd only mention the Dafny commit and date until we get a proper release.
Whatever you choose, please also replace the messages `Not used from 3.8 on` below.

Reply:
Cutting a Dafny PR to increment the minor version number so it'll be 3.8.0 at minimum